ariadne_django/formatters/errors/utils/get_postgres_error_details.py

```diff
@@ -0,0 +1,27 @@
+import django.core.exceptions
+from django.db import DatabaseError
+
+
+try:
+    import psycopg2  # pylint: disable=W0611
+except ImportError as error:
+    raise django.core.exceptions.ImproperlyConfigured("Cannot use this function without psycopg2.") from error
+
+
+def get_postgres_error_details(
+    error: DatabaseError,
+) -> dict:
+    error_code = getattr(getattr(error, "__cause__", None), "pgcode", None)
+    default_message = "A database error occurred that prevented this request from being completed."
+    if not error_code:
+        message = default_message
+    elif error_code.startswith("22") or (error_code.startswith("23") and error_code not in ["23505"]):
```

Comment:
can you convert these magic strings to named vars? I know this fn is small in scope, but `PG_INTEGRITY_CONSTRAINT_VIOLATION` is a lot clearer to me than `"23"`